tests/distributions/test_multivariate.py

```diff
@@ -2122,11 +2121,25 @@ def ref_rand(size, n, eta):
             size=1000,
         )
 
-    def test_default_transform(self):
+    @pytest.mark.parametrize(
+        argnames="shape, expected_shape",
+        argvalues=[
+            ((2,), ()),
+            pytest.param(
+                (3, 2),
+                (3,),
+                marks=pytest.mark.xfail(
+                    raises=NotImplementedError,
+                    reason="We do not support batch dimensions for pm.LKJCorr yet.",
+                ),
+            ),
+        ],
+    )
+    def test_default_transform(self, shape, expected_shape):
         with pm.Model() as m:
-            x = pm.LKJCorr("x", n=2, eta=1, shape=(3, 2))
+            x = pm.LKJCorr("x", n=2, eta=1, shape=shape)
         assert isinstance(m.rvs_to_transforms[x], MultivariateIntervalTransform)
-        assert m.logp(sum=False)[0].shape == (3,)
+        assert m.logp(sum=False)[0].type.shape == expected_shape
```

Comment:
No need to specify this manually:
```suggestion
        assert m.logp(sum=False)[0].type.shape == shape[:-1]
```

Reply:
The unused `expected_shape` should be removed as well